pkgs/development/libraries/ffmpeg-full/default.nix

```diff
@@ -239,11 +239,11 @@ assert opensslExtlib -> gnutls == null && openssl != null && nonfreeLicensing;
 
 stdenv.mkDerivation rec {
   pname = "ffmpeg-full";
-  version = "4.2.2";
+  version = "4.2.3";
 
   src = fetchurl {
-    url = "https://www.ffmpeg.org/releases/ffmpeg-${version}.tar.xz";
-    sha256 = "176jn1lcdf0gk7sa5l2mv0faqp5dsqdhx1gqcrgymqhfmdal4xfb";
+    url = "https://www.ffmpeg.org/releases/ffmpeg-${version}.tar.bz2";
```

Comment:
just curious, why did you change it to `bz2`?

Reply:
Reusing the hash from `ffmpeg_4` which uses `bz2`.

Not sure why they were using different tarballs, `ffmpeg-full` was changed to `xz` in 3a61de4e69431c667e6b900364a4229cd12f7019.